### PR TITLE
BAU: tweak README for Python 3.11

### PR DESCRIPTION
**Pull request by gidsg**
### Change description
- make python 3.11 requirement clearer
- Add .python-version for pyenv

- [ ] Unit tests and other appropriate tests added or updated **N/A**
- [x] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
Readme steps should work

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
+Python 3.11.2
+```
+
+
 Set up your virtual environment:
 ```
-python -m venv venv
-source /path/to/venv/bin/activate
+python -m venv .venv
+source .venv/bin/activate
```
Comment (cyrusdobbs) [inline README.md:18]: Could be worth noting that this is a unix only command and that to activate on Windows you have to run the `activate.bat` file in `venv/Scripts`

Comment (bobbi-hbrown) [reply README.md:18]: Maybe we should put the commands for both in here?

Comment (gidsg) [reply README.md]: Added in 95dbd8c

**Review by cyrusdobbs — APPROVED** [review]
LGTM 🥧

**Pull request merged**